src/app/authentication/forgot-password.component.ts

```diff
@@ -0,0 +1,38 @@
+import { Component, OnInit } from '@angular/core';
+import { NgForm } from '@angular/forms';
+import { AuthService } from '../services/auth.service';
+import { FormBuilder, FormGroup, Validators } from '@angular/forms';
+
+
+@Component({
+    selector: 'app-forgot-password',
+    templateUrl: './forgot-password.component.html',
+    styleUrls: ['./login.component.scss']
+})
+export class ForgotPasswordComponent {
+
+    resetForm: FormGroup;
+    errorMessage: String;
+    successMessage: String;
+
+    constructor(private _authService: AuthService, fb: FormBuilder) {
+        this.resetForm = fb.group({
+            'email' : [null, Validators.compose([Validators.required, Validators.pattern('[^ @]*@[^ @]*')])]
```

Comment:
did you run the linting file for typescript? and it's not complaining about the quotes around email?